TheMET/TheMET/DataModels/Art.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Art.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/03/2023.
+//
+
+import Foundation
+
+class Art: Decodable {
+    
+    let objectID: Int
+    
+    let primaryImage: String?
+    
+    let department: Department
```

Comment:
`Art` does not have direct information about its department. It has only department name as a `String`.

Reply:
Fixed